src/styles/components/doughnut-chart.scss

```diff
@@ -0,0 +1,85 @@
+@import "../settings";
+@import "../colours";
+@import "../common/utils";
+
+$border-width: 2px;
+
+@mixin doughnut-chart($remSize, $textRatio: 3) {
+  $size: remToPx($remSize/2);
+  overflow: hidden;
+  position: relative;
+  display: inline-block;
+  margin: 0 $global-margin/5;
+  width: $size * 2;
+  height: $size * 2;
+  border-radius: $size;
+  font-size: $remSize / $textRatio;
+  &__left-wrap {
+    overflow: hidden;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: $size;
+    height: $size * 2;
+    &__loader {
+      position: absolute;
+      left: 0;
+      top: 0;
+      border-radius: 1000px;
+      transform-origin: 0 50%;
+      left: $size;
+      width: $size;
+      height: $size * 2;
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+  }
+
+  &__right-wrap {
```

Comment:
A lot of `&__left-wrap` and `&__right-wrap` are in common - I wonder if we could create base style and extend them?